src/index.spec.js

```diff
@@ -722,7 +738,9 @@ describe('reactElementToJSXString(ReactElement)', () => {
 
   it('reactElementToJSXString(<div> {false} </div>)', () => {
     expect(reactElementToJSXString(<div> {false} </div>)).toEqual(
-      '<div>\n    \n</div>'
+      `<div>
+  {'  '}
+</div>`
```

Comment:
I do not like this output. But the previous one were also not pretty... so I guess we have to look at it in a dedicated PR.

Reply:
we can always provide an option to remove trailing whitespace no?